.github/workflows/npm-publish.yml

```diff
@@ -17,7 +17,7 @@ jobs:
       - name: Setup Node.js
         uses: actions/setup-node@v1
```

Comment:
I think these actions need to be updated to v3? I thought we ran into failures when using anything <v3 in a TDS recently. 

i.e.
uses: actions/setup-node@v3
uses:actions/checkout@v3

Reply:
i tried publishing with v1 yesterday via npm-beta-publish, and it seems that it went through even with v1 being referenced. I think we can update to v2 in the next release.
Ref GA: https://github.com/CMSgov/qpp-file-upload-api-client/actions/runs/3841387456